### PR TITLE
Fix contractions of it is in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by beraoudabdelkhalek**
Fix typos.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ If they don't help then please ask on the [User Group](https://groups.google.com
 ## Found an Issue?
 If you have found a bug then raise an issue on the zaproxy repo: https://github.com/zaproxy/zaproxy/issues
 
-Its worth checking to see if its already been reported, and including as much information as you can to help us diagnose your problem.
+Its worth checking to see if it's already been reported, and including as much information as you can to help us diagnose your problem.
```
Comment (kingthorin) [inline CONTRIBUTING.md:16]: May as well fix both.

```suggestion
It's worth checking to see if it's already been reported, and including as much information as you can to help us diagnose your problem.
```

**Comment by kingthorin** [comment]
Just put the text in a new comment.

**Comment by psiinon** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [5112d762\-481f\-4896\-99b7\-135cbac7d26b](https://eu.ast.checkmarx.net/projects/641c4d0f-0dae-4786-973b-ead9338157fd/scans?id=5112d762-481f-4896-99b7-135cbac7d26b&branch=beraoudabdelkhalek%2FCLA)
### No New Or Fixed Issues Found

**Comment by thc202** [comment]
Also amend your commit to match the changes.

**Comment by beraoudabdelkhalek** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by thc202 — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

**Pull request merged**